scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -73,4 +80,41 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
 
 }
 
-trait TestScalaJSModule extends ScalaJSModule with TestModule
\ No newline at end of file
+trait TestScalaJSModule extends ScalaJSModule with TestModule {
+  def scalaJSTestDeps = T {
+    Loose.Agg(
+      ivy"org.scala-js::scalajs-library:${scalaJSVersion()}",
+      ivy"org.scala-js::scalajs-test-interface:${scalaJSVersion()}"
+    )
+  }
+
+  def fastOptTest = T {
+    val bridge = scalaJSLinkerBridge(scalaJSLinkerClasspath().map(_.path))
+
+    val linkLibs = resolveDeps(scalaJSTestDeps)() ++ compileDepClasspath()
```

Comment:
Here we could be affected by this bug: https://github.com/lihaoyi/mill/issues/123
Cause `scalaJSTestDeps` resolved separately from `compileDepClasspath` they may contain `scalajs-library` and `scalajs-test-interface` of different version. They actually do, but I guess the first one(from `scalaJSTestDeps`) wins

Reply:
Yeah that's ok for now. We currently do not have any sort of clever classpath validation, so unfortunately things can be duplicated.